jboss/container/jolokia/bash/module.yaml

```diff
@@ -31,9 +31,9 @@ execute:
 - script: backward_compatibility.sh
 
 artifacts:
-#- name: jolokia-jvm-agent.jar
-- url: https://maven.repository.redhat.com/ga/org/jolokia/jolokia-jvm/1.5.0.redhat-1/jolokia-jvm-1.5.0.redhat-1-agent.jar
-  md5: d31c6b1525e6d2d24062ef26a9f639a8
+- target: jolokia-jvm-1.6.2-agent.jar
+  name: jolokia-jvm-1.6.2-agent.jar
```

Comment:
Shouldn't the name point to `-redhat` suffixed jar? Based on the following Cekit's [example](https://cekit.readthedocs.io/en/latest/handbook/redhat.html#artifacts).

Reply:
I think I agree; it will be clearer looking at the artefacts in an image whether or not it was a productized build JAR or not that way